app/src/androidTest/java/com/github/sdp/ratemyepfl/activity/MainActivityTest.kt

```diff
@@ -29,75 +37,79 @@ class MainActivityTest {
     @get:Rule(order = 1)
     val testRule = ActivityScenarioRule(MainActivity::class.java)
 
+    private lateinit var navController: TestNavHostController
 
-    @Test
-    fun navigateHomePageAddsTheCorrectFragment() {
-        onView(withId(R.id.activityMainBottomNavigationView)).perform(navigateTo(R.id.homeNavItem))
-        testRule.scenario.onActivity { activity ->
-            val fragment: Fragment? =
-                activity.supportFragmentManager.findFragmentById(R.id.mainActivityFragmentContainer)
-            assertEquals(true, fragment is HomeFragment)
-        }
-    }
-
-    @Test
-    fun navigateToReviewAddsTheCorrectFragment() {
-        onView(withId(R.id.activityMainBottomNavigationView)).perform(navigateTo(R.id.reviewNavItem))
-        testRule.scenario.onActivity { activity ->
-            val fragment: Fragment? =
-                activity.supportFragmentManager.findFragmentById(R.id.mainActivityFragmentContainer)
-            assertEquals(true, fragment is ReviewFragment)
-        }
-    }
-
-    @Test
-    fun navigateToEventPageAddsTheCorrectFragment() {
-        onView(withId(R.id.activityMainBottomNavigationView)).perform(navigateTo(R.id.eventNavItem))
-        testRule.scenario.onActivity { activity ->
-            val fragment: Fragment? =
-                activity.supportFragmentManager.findFragmentById(R.id.mainActivityFragmentContainer)
-            assertEquals(true, fragment is EventFragment)
-        }
-    }
-
-    @Test
-    fun navigateToMapPageAddsTheCorrectFragment() {
-        onView(withId(R.id.activityMainBottomNavigationView)).perform(navigateTo(R.id.mapNavItem))
-        testRule.scenario.onActivity { activity ->
-            val fragment: Fragment? =
-                activity.supportFragmentManager.findFragmentById(R.id.mainActivityFragmentContainer)
-            assertEquals(true, fragment is MapFragment)
-        }
-    }
-
-    @Test
-    fun testCourseButton() {
-        onView(withId(R.id.activityMainBottomNavigationView)).perform(navigateTo(R.id.reviewNavItem))
-        init()
-        onView(withId(R.id.courseTabButton)).perform(click())
-        intended(toPackage("com.github.sdp.ratemyepfl"))
-        release()
-    }
+    @Before
+    fun setup() {
```

Comment:
For this file, you should at least try to run the activity and test the navigation. Just testing the fragments in isolation doesn't mean the app is working.

Reply:
Yes of course, those were commented and I forgot to put them back. I'll add them.